packages/design-system/src/components/DateField/DateField.tsx

```diff
@@ -150,6 +154,32 @@ export interface DateFieldProps {
   yearValue?: DateFieldYearValue;
 }
 
-export default class DateField extends React.Component<DateFieldProps, any> {
-  render(): JSX.Element;
+export function DateField(props: DateFieldProps): React.ReactNode {
```

Comment:
Change to return type React.Element
`export function DateField(props: DateFieldProps): React.ReactElement {`